src/code/gfxalloc.c

```diff
@@ -1,14 +1,37 @@
 #include "global.h"
 
-Gfx* Gfx_Open(Gfx* gfx) {
-    return gfx + 1;
+/**
+ * Creates a new temporary graphics display list pointer, using the memory reserved by gfxDisp
+ *
+ * @param gfxDisp is the display list yielding memory. It cannot be written to until Gfx_Close is called.
+ * @returns a new graphics display list pointer.
+ *
+ * @note This is used to give WORK_DISP more memory to write instructions without increasing the WORK_DISP buffer size.
```

Comment:
```suggestion
 * @note This may be used to reserve memory in a larger display buffer to be called from a small display buffer. For
 * example, space in POLY_OPA can be reserved to be called from the WORK and OVERLAY buffers through this function.
```
Before this was too specific since it's also commonly used for OVERLAY.